shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationAttempts.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.di.authentication.shared.entity;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class AuthenticationAttempts {
+
+    private String attemptIdentifier;
+    private String email;
```

Comment:
@dbes-gds, shall we remove email from this? I see it's proposed as a nice addition but shall we delay adding it until we see we need it?

Reply:
yes can remove it, the idea was to make testing a bit easier as you can see which emails have counts, but can just use the internal subject id.